packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/layout_explorer_widget.dart

```diff
@@ -275,14 +278,14 @@ abstract class LayoutExplorerWidgetState<W extends LayoutExplorerWidget,
 
   // TODO(albertusangga): Investigate why onForceRefresh is not getting called.
   @override
-  Future<Object> onForceRefresh() async {
+  Future<Object?> onForceRefresh() async {
     _setProperties(await fetchLayoutProperties());
     return null;
   }
 
   /// Currently this is not working so we should listen to controller selection event instead.
   @override
-  Future<void> onInspectorSelectionChanged() {
+  Future<void>? onInspectorSelectionChanged() {
```

Comment:
both of these should just have the return type `Future<void>` and not add a return statement

Reply:
done